EDDiscovery/Translations/portugues-pt-br.tlf

```diff
@@ -582,20 +582,20 @@ SetNoteForm.label_index: "Set Note" => "Imposta Nota"
 /////////////////////////////////////////////////////////////// MessageBox
 
 MessageBoxTheme.Warning: "Warning" => "Atenção"
-.Ignore: "Ignore" => "Ignora"
-.Retry: "Retry" => "Retenta"
-.Abort: "Abort" => "Aborta"
-.Cancel: "Cancel" => "Cancela"
+.Ignore: "Ignore" => "Ignorar"
+.Retry: "Retry" => "Retentar"
+.Abort: "Abort" => "Abortar"
+.Cancel: "Cancel" => "Cancelar"
```

Comment:
Apesar de estar correto, não entendo porque decidiu e também não tenho certeza se seria o caso de mudar o tempo verbal. Em inglês, estamos usando o tempo comum para as ações, o _Simples Present_ - que corresponde ao _Presente do Indicativo_ em Português - e você está traduzindo tudo no Infinitivo. Como disse, apesar de ser comumente usado, não seria a forma mais apropriada de tradução nesse contexto... por exemplo: "Ignorar", em Inglês, seria "To Ignore", e não "Ignore". Enfim, no fundo é apenas uma questão estética...